build/PackageInfo.xml

```diff
@@ -0,0 +1,6 @@
+<pkg-info format-version="2" identifier="io.lbry.LBRY.pkg" version="0.10.0" install-location="/" auth="root">
+  <payload installKBytes="173006" numberOfFiles="3608"/>
```

Comment:
how did you calculate the number of files and the installation size? we can't have this hardcoded, since it will need to change anytime we make code changes